tests/bsim/bluetooth/audio_samples/bap_broadcast_sink/compile.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+# Copyright 2023 Nordic Semiconductor ASA
```

Comment:
Nitpick, but the year should be 2024

Reply:
Maybe? :D We don't have any guidelines on how/when the copyright years should be updated, and since this code was just copied from another file with that year, then I really don't know :D